docs/reference/aggregations/metrics/extendedstats-aggregation.asciidoc

```diff
@@ -116,4 +116,26 @@ It turned out that the exam was way above the level of the students and a grade
         }
     }
 }
---------------------------------------------------
\ No newline at end of file
+--------------------------------------------------
+
+==== Missing value
+
+The `missing` parameter defines how documents that miss a value should be treated.
+By default they will be ignored but it is also possible to treat them as if they
+had a value.
+
+[source,js]
+--------------------------------------------------
+{
+    "aggs" : {
+        "grades_stats" : {
+            "extended_stats" : {
+                "field" : "grade",
+                "missing": 0 <1>
+            }
+        }
+    }
+}
+--------------------------------------------------
+
+<1> Document that don't have a value for the `grade` field will be treated as if they had `0` as a value
```

Comment:
Documents without a value in the `grade` field will fall into the same bucket as documents that have the value `0`.